src/utils.h

```diff
@@ -5,6 +5,7 @@
 #include <cstring>
 #include <sstream>
 #include <string>
+#include <tuple>
```

Comment:
I can't say that I completely understand why the need for this was surfaced by the changes in the content and usage of `r_utils.h`. But I'm also not sure that I really need to get to the bottom of it.